lib/bigcommerce.js

```diff
@@ -142,9 +142,20 @@ BigCommerce.prototype.authorise = function(query, cb) {
 };
 
 BigCommerce.prototype.createAPIRequest = function() {
+  var accept = 'application/json';
+  switch (this.config.responseType) {
+    case 'xml':
+      accept = 'text/xml';
+      break;
+    case 'json':
```

Comment:
You don't need this since you already set `var accept = 'application/json';` above